### DISCUSSION TITLE
Can not commit changes in git hub

### DISCUSSION THREAD

**Discussion by viryakho** [community/community#29356] [New to GitHub]
Why when I want commit changes in new file, the button can't click.

**Comment by samuel-ma** [comment]
Are you committing to the new file from vscode or directly from github

**Reply by viryakho** [reply]
directly from github

**Reply by MadhushaPrasad** [reply]
Is that shown as a GPG keys issue? can I see the error of the console show?

**Reply by viryakho** [reply]
it doesnt show anything
